### DISCUSSION TITLE
Separate director for 'db_dir'

### DISCUSSION THREAD

**Discussion by cani1989** [guydavis/machinaris#843] [Q&A]
**Describe the request**
If a path is defined as `db_dir` then put the Chia network database in here. 

Reason for this is that on unRAID I am backing up my `appdata` folder and I do not wish to backup the database every time as it takes too much space.

**Comment by guydavis** [comment]
Greetings! Yes, the Chia blockchain database itself is well over 100 GB in size now, even the smaller v2 version.  Here are my suggestions for you:

1. Please ensure you have performed the `chia db upgrade` that [migrates](https://github.com/guydavis/machinaris/wiki/Chia#database-upgrade) from the even larger v1 database to ~110 GB v2 Chia database.
2. In the Unraid admin UI, choose "Settings", then "Backup/Restore Appdata" (plugin link at bottom of settings page).  This plugin offers a 'Excluded Folder Browser' multi-checkbox list.  Check (enable) the 'Machinaris' container to exclude it from appdata backup.
3. Alternatively, you can move the Machinaris appdata folder to an Unassigned Drive (must be SSD) within the Unraid Docker configuration for Machinaris.  Stop the container, Change the host Path from `/mnt/user/appdata/machinaris` to `/mnt/disks/MY_FAST_SSD/machinaris`, then move the appdata directory over to the new disk and start the up the Machinaris container.

Sorry, I'd hoped to paste screenshots of my settings to illustrate, but my Unraid server has been locking up recently, requiring reboots.  Currently down.  I will add them if I can get it running again.  Hope this helps.

EDIT: Some screenshots of above suggestions:
![image](https://user-images.githubusercontent.com/12936/214054244-bee06871-d08e-4803-a1a8-6f80388bbe20.png)

![image](https://user-images.githubusercontent.com/12936/214054403-72891ba8-4d55-4427-9a42-bf02ef727d94.png)
